src/configs.test.ts

```diff
@@ -0,0 +1,39 @@
+import {readdirSync} from 'fs';
+import {join} from 'path';
+
+import {Config, Variant} from './types';
+
+describe('configs', () => {
+  const configsPath = join(__dirname, 'configs');
+  const configFileNames = readdirSync(configsPath);
+
+  it.each(configFileNames)('%s', (configFileName) => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const config: Config = require(`configs/${configFileName}`).config;
+    const variantsByFigmaName: Record<
+      string,
+      {property: string; variant: Variant}
+    > = {};
+
+    for (const [property, variants] of Object.entries(config)) {
```

Comment:
I don't like the name `property` here, but I don't know what else to call the top-level keys of the `Config` object ("surface", "onSurface", etc.). Any suggestions?

Reply:
Hmm... roles? https://vault.shopify.io/pages/4806-Color-roles-and-variants

Reply:
That's a great suggestion. Thanks for the link to that page, I hadn't seen it before.
I'm wondering, though, if these properties will _always_ be color roles or if they could also relate to spacing, typography, etc.I couldn't find any clarity in [the Theo documentation](https://github.com/salesforce-ux/theo) about it, but I admit that the content there was way over my head.
Maybe @kaelig can weigh in on this.

Reply:
`property` works! (we don't have a generic name for it in our types that would work for non-color tokens, so I think it's fair to go with the Theo vocabulary)